python/samples/core_chess_game/main.py

```diff
@@ -20,10 +22,15 @@
 from autogen_core.tools import FunctionTool
 from chess import BLACK, SQUARE_NAMES, WHITE, Board, Move
 from chess import piece_name as get_piece_name
-from common.agents._chat_completion_agent import ChatCompletionAgent
-from common.patterns._group_chat_manager import GroupChatManager
-from common.types import TextMessage
-from common.utils import get_chat_completion_client_from_envs
+from patterns import GroupChatManager
+from pydantic import BaseModel
+from utils import get_chat_completion_client_from_envs
```

Comment:
We really should update to not use this...

Reply:
Makes sense. Will update to use the component config?

Reply:
I did it for magentic one here: https://github.com/microsoft/autogen/pull/4928